test-site/src/non-real-time-test.ts

```diff
@@ -28,7 +41,18 @@ let showMs = true
   }
 }
 ;(window as any).testNonRealTime = async () => {
-  const myvad = await NonRealTimeVAD.new()
+  // Configure VAD based on environment
+  const vadConfig = process.env.NODE_ENV === 'development' 
+    ? {
+        // Use local assets in development
+        modelURL: "./silero_vad_legacy.onnx",
+        modelFetcher: (path: string) => fetch(path).then(r => r.arrayBuffer())
+      }
+    : {
+        // Use default CDN assets in production
+      }
```

Comment:
I needed these to get my local changes to work (I think, i was kidna confused) but these may not be necessary, happy to remove them.